rad/src/operators/array.rs

```diff
@@ -91,7 +99,7 @@ pub fn filter(input: &RadonArray, args: &[Value]) -> Result<RadonTypes, RadError
     let first_arg = args.get(0).ok_or_else(wrong_args)?;
     match first_arg {
         Value::Integer(arg) => {
-            let filter_code = RadonFilters::from_i128(*arg).ok_or_else(wrong_args)?;
+            let filter_code = RadonFilters::try_from(*arg as u8).map_err(|_| wrong_args())?;
```

Comment:
The conversion to `u8` should also be a try_from:
```suggestion
            let filter_code = RadonFilters::try_from(u8::try_from(*arg)?).map_err(|_| wrong_args())?;
```